cairo-oracle-hint-processor/src/rpc_hint_processor.rs

```diff
@@ -109,74 +107,143 @@ impl<'a> Rpc1HintProcessor<'a> {
         );
         println!("let the oracle decide... Inputs: {data:?}");
 
-        let client = reqwest::blocking::Client::new();
+        let client = reqwest::blocking::ClientBuilder::new()
+            .timeout(Duration::from_secs(10)) // Short timeout for each request

```

Comment:
Should we allow the user to also change this?

Reply:
Yes good point I can parametize as well